Sprint-2/implement/vat.js

```diff
@@ -8,3 +8,14 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+function fullPriceWithVAT(noVatValue) {
+    const VAT = 1.2;
+
+    return `R${noVatValue * VAT}`;
```

Comment:
Why did you decide to return a string with a currency symbol in it instead of returning the full price as a number?

If you were to design the function so that it has a higher chance to be reused (for various purposes), which type of return value do you think is better?

Reply:
Yes, it would be more wise to return a number than a string with a currency for reusability. I've adjusted that code